src/main/java/org/folio/helper/BindHelper.java

```diff
@@ -57,7 +66,9 @@ public Future<ReceivingResults> bindPieces(BindPiecesCollection bindPiecesCollec
   private Future<ReceivingResults> processBindPieces(BindPiecesCollection bindPiecesCollection, RequestContext requestContext) {
     // 1. Get piece records from storage
     return retrievePieceRecords(requestContext)
-      // 2. Update piece isBound flag
+      // 2. Check if there are any outstanding requests for items
+      .compose(piecesGroupedByPoLine -> checkRequestsForPieceItems(piecesGroupedByPoLine, bindPiecesCollection, requestContext))
+      // 3. Update piece isBound flag
       .map(this::updatePieceRecords)
       // 3. Update currently associated items
```

Comment:
comments should be update, so it should be 4

Reply:
Forgot to do it after rebasing, fixed now